fenix/app/src/main/res/xml/tracking_protection_preferences.xml

```diff
@@ -8,9 +8,10 @@
         android:layout="@layout/tracking_protection_learn_more_preference"
         android:summary="@string/preference_enhanced_tracking_protection_explanation_2"
         app:allowDividerBelow="false" />
-    <SwitchPreference
+    <CheckBoxPreference
```

Comment:
Note: I was unable to override the SwitchPreference layout expected - the Switch widget was not responsive to using existing ID references. Tested with:
- @android:id/checkbox
- @android:id/switchInputMethod
- @android:id/switch_widget
- @android:id/toggle
Hence this is now a CheckBoxPreference